client/my-sites/email/email-forwarding/index.jsx

```diff
@@ -34,46 +37,61 @@ class EmailForwarding extends Component {
 	static propTypes = {
 		emailForwards: PropTypes.array,
 		emailForwardingLimit: PropTypes.number.isRequired,
+		emailForwardingType: PropTypes.string,
 		selectedDomainName: PropTypes.string.isRequired,
 		siteSlug: PropTypes.string.isRequired,
 		translate: PropTypes.func.isRequired,
 	};
 
 	render() {
-		const { emailForwards, selectedDomainName } = this.props;
+		const { selectedDomainName, translate } = this.props;
 		return (
-			<div className="email-forwarding">
+			<Main>
 				<QueryEmailForwards domainName={ selectedDomainName } />
-				{ emailForwards ? this.renderMain() : this.renderPlaceholder() }
-			</div>
+				<Header onClick={ this.goToEditEmail } selectedDomainName={ selectedDomainName }>
+					{ translate( 'Email Forwarding' ) }
+				</Header>
+				{ this.renderContent() }
+			</Main>
 		);
 	}
 
-	renderPlaceholder() {
-		return <MainPlaceholder goBack={ this.goToEditEmail } />;
+	renderContent() {
+		const { emailForwardingType, selectedDomainName, siteSlug } = this.props;
+		switch ( emailForwardingType ) {
+			case 'forward':
+				return this.renderForwards();
+			case 'custom':
+				return <EmailForwardingCustomMxList selectedDomainName={ selectedDomainName } />;
+			case 'google-apps':
+				return (
+					<EmailForwardingGSuiteDetails
+						selectedDomainName={ selectedDomainName }
+						siteSlug={ siteSlug }
+					/>
+				);
+			case 'google-apps-another-provider':
+				return <EmailForwardingGSuiteDetailsAnotherProvider />;
+			case null:
+			default:
+				return <EmailForwardingPlaceholder />;
+		}
```

Comment:
That might just be me but I find this `switch` block difficult to read. Adding a few blank lines would certainly help addressing that.